phone/features/traveling/src/main/java/jp/ac/mayoi/traveling/TripCancelButton.kt

```diff
@@ -0,0 +1,42 @@
+package jp.ac.mayoi.traveling
+
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import jp.ac.mayoi.core.resource.MaigoCompassTheme
+import jp.ac.mayoi.core.util.MaigoButton
+
+@Composable
+fun TripCancelButton(
+    onTripCancelButtonClick: () -> Unit
+) {
+    MaigoButton(
+        onClick = onTripCancelButtonClick,
+        colors = ButtonDefaults.buttonColors(containerColor = Color(0xFFB3261E)),
+        modifier = Modifier
+            .width(200.dp),
+    ) {
+        Text(
+            fontWeight = FontWeight.Bold,
+            text = "旅行を中断",
+            fontSize = 16.sp
+        )
+    }
+}
+
+@Preview(showBackground = true)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mayoi-design/maigo_compass_android/blob/6a718ee0c504aa5c11220921c2536f827a971938/phone/features/traveling/src/main/java/jp/ac/mayoi/traveling/TripCancelButton.kt#L34"></span>Composables annotated with @Preview that are used only for previewing the UI should not be public.See https://slackhq.github.io/compose-lints/rules/#preview-composables-should-not-be-public for more information.</td>
    </tr>
  </tbody>
</table>